site/docs/intro/getting-started.mdx

```diff
@@ -66,7 +67,9 @@ Preevy is designed to be easy to use, with an API smiliar to [Docker Compose](ht
   ```bash
   preevy up
   ```
-<br />
+
+**Note:** Make sure the repo/project you are working and running the `preevy up` command is a git initialized repo and at least one commit has been made. This is required for Preevy to create a unique preview environment name.
```

Comment:
This phrasing may create the impression that git is required in order to use Preevy. Also, it would be helpful to state the reason for having a git context.

```suggestion
**Note:** Preevy uses the git repo at the current directory to calculate a stable environment ID for your project. Make sure a git repo is initialized (with at least one commit), or add the `--id` flag to explicitly specify the environment ID.
```